com.unity.render-pipelines.high-definition/Runtime/Material/Lit/Lit.hlsl

```diff
@@ -2005,6 +2005,15 @@ void PostEvaluateBSDF(  LightLoopContext lightLoopContext,
 #else
     GetScreenSpaceAmbientOcclusionMultibounce(posInput.positionSS, preLightData.NdotV, bsdfData.perceptualRoughness, bsdfData.ambientOcclusion, bsdfData.specularOcclusion, bsdfData.diffuseColor, bsdfData.fresnel0, aoFactor);
 #endif
+
+#if (SHADERPASS == SHADERPASS_DEFERRED_LIGHTING || SHADERPASS == SHADERPASS_FORWARD) && !defined(_SURFACE_TYPE_TRANSPARENT)
```

Comment:
Marker for Seb.
Note: we should move this code to lightloop.hlsl
just after this one:
        // Note: builtinDataProbeVolumes.bakeDiffuseLighting and builtinDataProbeVolumes.backBakeDiffuseLighting were combine inside of ModifyBakedDiffuseLighting().
        builtinData.bakeDiffuseLighting += builtinDataProbeVolumes.bakeDiffuseLighting;

Let's discuss